<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlatFutures SmartMirror</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="captureButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analysisButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundImg">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>300</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/face.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="TL_Buttion">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#TL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TC_Button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#TC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TR_Button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#TR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ML_Button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#ML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MC_Button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#MC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MR_Button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#MR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BL_Button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#BL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BC_Button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#BC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BR_Button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#BR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modelButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>CAM ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="camView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logoImg">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>660</y>
      <width>300</width>
      <height>72</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
